### ISSUE TITLE
[a11y]: Export Modal, Exporting file status message not conveyed completely by the Screen reader

### ISSUE THREAD

**Issue by devadula-nandan** [carbon-design-system/ibm-products#6020]
### Package

@carbon/ibm-products

### Browser

Firefox

### Operating System

Windows

### Package version

v2.49.0-rc.0

### Automated testing tool and ruleset

manual

### Assistive technology

JAWS

### Description

Reported by Raghu during QA Audit.

The "Exporting file" status is not conveyed to the screen reader user, instead, it announces as "Loading", also once the process is done, the screen reader does not announce the success status visually populated as "The file has been exported.".

### Recommended fix
Any dynamic status indication must be presented to screen reader users instantly, or else the same group may not come to know about such status info. usage of aria-live was recommended on the call.

### WCAG 2.1 Violation

4.1.3 Status Messages

### Reproduction/example

https://ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-export-exportmodal--standard&globals=viewport:basic

### Steps to reproduce

open the export modal and click on export button
observe the live status is not conveyed to the user

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems
